x-pack/test/api_integration/apis/security_solution/timeline.ts

```diff
@@ -6,135 +6,54 @@
  */
 
 import expect from '@kbn/expect';
+import {
+  SavedTimeline,
+  TimelineType,
+} from '../../../../plugins/security_solution/common/types/timeline';
 
-import { Direction } from '../../../../plugins/security_solution/common/search_strategy';
-// @ts-expect-error
-import { timelineQuery } from '../../../../plugins/security_solution/public/timelines/containers/index.gql_query';
-// @ts-expect-error
-import { GetTimelineQuery } from '../../../../plugins/security_solution/public/graphql/types';
 import { FtrProviderContext } from '../../ftr_provider_context';
+import { createBasicTimeline, createBasicTimelineTemplate } from './saved_objects/helpers';
 
-const TO = '3000-01-01T00:00:00.000Z';
-const FROM = '2000-01-01T00:00:00.000Z';
+export default function ({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
 
-// typical values that have to change after an update from "scripts/es_archiver"
-const DATA_COUNT = 2;
-const HOST_NAME = 'suricata-sensor-amsterdam';
-const TOTAL_COUNT = 96;
-const EDGE_LENGTH = 2;
-const CURSOR_ID = '1550608949681';
+  describe('Timeline', () => {
+    it('Make sure that we get Timeline data', async () => {
+      const titleToSaved = 'hello timeline';
+      await createBasicTimeline(supertest, titleToSaved);
 
-const FILTER_VALUE = {
```

Comment:
I think we should still test the API with filter

Reply:
yes, The test was more like targeting to querying events, I rewrote this test to make it fit for timeline api. I'll add another test for events in a follow up PR as this one is a big one already.

Reply:
https://github.com/elastic/kibana/pull/97519